torch/_inductor/fx_passes/post_grad.py

```diff
@@ -795,8 +795,16 @@ def can_inplace(node, mutated_arg):
                     graph.erase_node(copy_node)
 
                 node.target = inplaceable_op.inplace_op
-    for node, replacement in replace_list:
-        node.replace_all_uses_with(replacement)
+
+    def get_replacement(node):
+        ori_node = node
+        while node in replace_dict:
+            node = replace_dict[node]
+        replace_dict[ori_node] = node
+        return node
```

Comment:
Nit: as this is done only once for every distinct key node in the `replace_dict`, I'm wondering if we should just inline this in the for-loop below. Then also could start from the `replace_dict[node]` to save one iteration and skip the assignment as the same `replace_dict[node]` won't be queried again.